### ISSUE TITLE
FreeBSD packages are of wrong architecture

### ISSUE THREAD

**Issue by ByJumperX4** [Jelmerro/Vieb#178]
**Checklist**
- [x] I have read the [frequently asked questions](https://github.com/Jelmerro/Vieb/blob/master/FAQ.md)
- [x] I did not find any similar issues in the [issue tracker](https://github.com/Jelmerro/Vieb/issues)

**Describe the bug**
The package for FreeBSD is of wrong architecture:
```
byjumperx4@PCmoncamo ~/Downloads> doas pkg add vieb-4.5.1.freebsd
Installing vieb-4.5.1...
pkg: wrong architecture: Linux:5:x86:64 instead of FreeBSD:13:amd64

Failed to install the following 1 package(s): vieb-4.5.1.freebsd
```

**To Reproduce**
Get a FreeBSD install, download official FreeBSD vieb binary, try to install it with `pkg add vieb-x.x.x.freebsd`

**Expected behavior**
I expect the package to be installed without problem.

**Specs**
 - OS: FreeBSD 13.0-RELEASE amd64
 - Version: Vieb 4.5.1

**Comment by Jelmerro** [comment]
I have no clue where to start fixing this, there are a number of open issues related to freebsd in both electron and the electron-bulder repositoy, so it might not even be a Vieb specific problem. Though I can't tell, because I have never once touched freebsd. Is this an error you have ever had for other packages? And can you run other Electron-based apps providing a similar package?

**Comment by Jelmerro** [comment]
Does it work when you force the installation with `-f`? If not, is the error the same?

**Comment by Jelmerro** [comment]
I would try toe `-f` option to force the installation, but it seems that freebsd is just a wrapped linux app for now. Below are some related issues and questions regarding proper freebsd support.
https://github.com/electron-userland/electron-builder/issues/3729
https://github.com/electron-userland/electron-builder/issues/1470
https://github.com/electron/electron/issues/3797
https://stackoverflow.com/questions/65295127/freebsd-linux-target-in-electron-is-it-possible-to-use-the-generated-file-with

**Comment by ByJumperX4** [comment]
FreeBSD only haves electron 11 ported, 12 is not ported. And a linux build won't work on FreeBSD: FreeBSD is not Linux. Maybe with the linuxulator, but that would be a pain.

**Comment by Jelmerro** [comment]
I know that FreeBSD isn't Linux, but that's what electron-builder makes of it. It's a Linux app that's wrapped as a FreeBSD package, which [could be made compatible](https://docs.freebsd.org/en/books/handbook/linuxemu/), but the target `freebsd` doesn't do anything to make this actually work. I tested this with a FreeBSD VM as well, and it's just a Linux package that's been compressed with XZ. There's no way this will work of course, and the linuxemu keeps complaining about various missing `.so` files, so I gave up on that for now. I apologize for giving the impression that Vieb had a FreeBSD package all this time, but clearly FreeBSD isn't supported by Electron or Electron Builder at all (see the links from my previous comment). I'll have removed it from the download page to avoid any future confusion.

If at all possible I encourage others to make a proper Vieb release for the various BSD variants. It seems that once a ports package is added, it should automatically be listed in the third-party releases badge of [repology](https://repology.org/) (see footnotes for supported repos).
